demisto_sdk/commands/unify/integration_script_unifier.py

```diff
@@ -163,12 +164,34 @@ def unify(self, file_name_suffix=None):
                 yml_unified = self.add_contributors_support(yml_unified, contributor_type, contributor_email,
                                                             contributor_url, author)
 
+        if self.test:
+            yml_unified = self.add_test(yml_unified)
+
         output_map = self.write_yaml_with_docker(yml_unified, self.yml_data, script_obj)
         unifier_outputs = list(output_map.keys()), self.yml_path, script_path, image_path, desc_path
         print_color(f'Created unified yml: {list(output_map.keys())}', LOG_COLORS.GREEN)
 
         return unifier_outputs[0]
 
+    def add_test(self, unified_yml):
```

Comment:
- Let’s change the name to something more informative (maybe something like `add_test_section`)
- please add type hints
- this method can be static
```suggestion
    @staticmethod
    def add_test(unified_yml: Dict) -> Dict:
```

Reply:
Done